src/cli/telemetry/test/telemetry.spec.ts

```diff
@@ -76,3 +76,132 @@ describe('checkTelemetry', () => {
     expect(await telemetry.checkTelemetry()).toBe(true);
   });
 });
+
+describe('hasAppTarget', () => {
+
+  beforeEach(() => {
+    getStencilCLIConfig()?.resetInstance();
+
+    initializeStencilCLIConfig({
+      sys: createSystem(),
+      logger: mockLogger(),
+      flags: { ci: false },
+      validatedConfig: { config: { outputTargets: [] } } as LoadConfigResults,
+      args: [],
+    });
+  });
+
+
+  it('Result is correct when outputTargets are empty', () => {
+    getStencilCLIConfig().validatedConfig = { config: { outputTargets: [] } } as LoadConfigResults
+    expect(telemetry.hasAppTarget()).toBe(false)
+  });
+
+  it('Result is correct when outputTargets contains www with no baseUrl or serviceWorker', () => {
+    getStencilCLIConfig().validatedConfig = { config: { outputTargets: [{ type: "www" }] } } as LoadConfigResults;
+    expect(telemetry.hasAppTarget()).toBe(false)
+  });
+
+  it('Result is correct when outputTargets contains www with default baseUrl value', () => {
+    getStencilCLIConfig().validatedConfig = { config: { outputTargets: [{ type: "www", baseUrl: "/" }] } } as LoadConfigResults;
+    expect(telemetry.hasAppTarget()).toBe(false)
+  });
+
+  it('Result is correct when outputTargets contains www with serviceWorker', () => {
+    getStencilCLIConfig().validatedConfig = { config: { outputTargets: [{ type: "www", serviceWorker: { swDest: "./tmp" }, }] } } as LoadConfigResults
+    expect(telemetry.hasAppTarget()).toBe(true)
+  });
+
+  it('Result is correct when outputTargets contains www with serviceWorker', () => {
```

Comment:
I think this test name needs to be updated, there's no service worker field set in this test